Remotion/Web/Core/UI/Controls/WebTreeView.cs

```diff
@@ -789,6 +795,34 @@ private void RenderNodeLabel (HtmlTextWriter writer, WebTreeNode node, string no
       writer.RenderEndTag();
     }
 
+    /// <summary> Renders the <paramref name="node"/>'s badge (if there is one) onto the <paremref name="writer"/>. </summary>
+    private void RenderNodeBadge (HtmlTextWriter writer, WebTreeNode node)
+    {
+      var badge = node.Badge;
+      if (badge == null || badge.Value == string.Empty)
```

Comment:
```suggestion
      if (string.IsNullOrEmpty (badge?.Value))
```
It feels more consistent this way. Mostly, when compared to the other usages.